ethereum/statetransition/src/test/java/tech/pegasys/teku/statetransition/forkchoice/ForkChoiceTest.java

```diff
@@ -838,9 +838,7 @@ void onAttestation_shouldBeInvalidWhenInvalidCheckpointThrown() {
     assertThat(result)
         .isCompletedWithValue(
             AttestationProcessingResult.invalid(
-                String.format(
-                    "Checkpoint state (%s) must be at or prior to checkpoint slot boundary (%s)",
-                    targetBlock.getSlot(), targetCheckpoint.getEpochStartSlot(spec))));
+                "LMD vote must be consistent with FFG vote target"));
```

Comment:
We don't pick up this error as part of requesting the state anymore - it gets picked up by the actual validation process.  So same rule applied but it gets the rejection message from the spec rather than something specific to getting the checkpoint state.